usr/src/uts/common/brand/lx/syscall/lx_fcntl.c

```diff
@@ -437,6 +444,9 @@ lx_fcntl_lock(int fd, int lx_cmd, void *arg)
 	case F_GETLK:
 	case F_SETLK:
 	case F_SETLKW:
+	case F_OFD_GETLK:
+	case F_OFD_SETLK:
+	case F_OFD_SETLKW:
```

Comment:
NOTE:  OFD ops are only allowable on 64-bit or with *large file* support.  For those you'll need to use the 64-bit.  You'll need to have to make checks for F_OFD_* here as it is ONLY supported in 32-bit apps on LARGE FILE ops.  (F_OFD_* is implicitly F_OFD_*64.)  Basically the 32-bit struct size won't work for F_OFD_*.

Reply:
I've determined this is a big mess, and the code-changes underneath here will add more mess.  If I can push to this PR, it'll be clear how and why.

Reply:
Now in #471